MobilePlatformServices/Sources/MobilePlatformServices/App Info/Response Objects/App.swift

```diff
@@ -5,25 +5,31 @@ public struct App: Decodable {
     public let minimumVersion: Version
     public let allowAppUsage: Bool
     public let releaseFlags: [String: Bool]
+    public let featureFlags: [String: Bool]
```

Comment:
A thought for the future, it feels like some of this PR (for example: this addition of feature flags) could / should have been completed in another pull request and merged in separately 🙂 just to keep our changes smaller. That makes it easier to revert if we break anything.

Reply:
Agreed, I think the ticket for the feature flag change as a specific piece of work got closed.
Happy to break this stuff into a separate PR and get that merged before this. I started pulling on the AppEnvironment stuff as a piece of string and led to a lot of cleaning up in there and the configuration files, as I say, happy to take that stuff out and put in a separate PR if we think this is too much for one PR 👍

Reply:
I think leave it for now as it's probably more complex to split it up, but one for us to be careful of next time 🙂